app/services/notices/setup/initiate-session.service.js

```diff
@@ -45,25 +46,30 @@ async function go(notificationType, monitoringStationId = null) {
     .insert({
       data: {
         ...additionalData,
-        ...noticeType
+        ...notice,
+        journey
       }
     })
     .returning('id')
 
   return {
     sessionId: session.id,
-    path: `${redirectPath}`
+    path: _redirect(journey)
   }
 }
 
-async function _adHoc() {
-  const session = await SessionModel.query().insert({ data: {} }).returning('id')
-
-  return {
-    sessionId: session.id,
-    path: `licence`
+function _redirect(journey) {
```

Comment:
I would suggest refactoring this as

```javascript
function _redirect(journey) {
  if (journey === 'standard') {
    return 'returns-period'
  }

  if (journey === 'standard') {
    return 'returns-period'
  }

  return 'licence'
}
```

> Not a suggestion because review UI wouldn't let me highlight all the relevant lines.